src/pkl/index.pkl

```diff
@@ -16,7 +16,6 @@
 
 output {
   files {
-    ["out/pkl.tmLanguage.json"] = import("pkl.tmLanguage.pkl").output
-    ["out/pkl.snippets.json"] = import("snippets.pkl").output
+    ["out/pkl.snippets.json"] = import("../../textmate-bundle/pkl/snippets.pkl").output
```

Comment:
For consistency sake, how about let's keep both of these here?

Sidenote: wouldn't be a bad idea to turn these into packages.

Reply:
You mean generate `pkl.tmLanguage.json` here from the source pkl instead of using the pre-rendered textmate language?

Reply:
Yup